### PR TITLE
 Extract script for nearlib testing from .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

File: scripts/test_nearlib.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+mkdir new_project
+cd new_project
+npm install near-shell
+node_modules/near-shell/near new_project
+npm install
+npm run build
+npm test
+cd ..
+
+./scripts/kill_devnet.sh
```
Comment (evgenykuzyakov) [inline scripts/test_nearlib.sh]: kill_devnet.sh is not working. no need to include it unless we fix it :)
It also breaks CI if you fix it

Comment (janedegtiareva) [reply scripts/test_nearlib.sh]: oh lol when did it get broken? Also tests used to fail if devnet was not killed, so it's surprising

Comment (evgenykuzyakov) [reply scripts/test_nearlib.sh]: I don't think it ever worked since it doesn't have run env specified, unless it guesses it from the parent script.

Comment (vgrichina) [reply scripts/test_nearlib.sh]: @evgenykuzyakov what is this then?

```
+ ./scripts/kill_devnet.sh
./scripts/test_nearlib.sh: line 27:    11 Killed                  ./target/release/devnet
```

https://gitlab.com/nearprotocol/nearcore/-/jobs/184784258

Comment (evgenykuzyakov) [reply scripts/test_nearlib.sh]: *magic*

**Pull request merged**